### ISSUE TITLE
centered rmsprop

### ISSUE THREAD

**Issue by alirezag** [torch/optim#157]
I slightly modified the rmsprop code to accommodate the centered version which converges faster. Is it okay to merge it?

**Comment by soumith** [comment]
if it is guarded behind a kwarg, we might consider

**Comment by alirezag** [comment]
Yes, it uses the config argument of the standard rmsprop and leaves the normal behavior intact if not set.
